app/src/components/main/analytics/events/profilePageEvents.js

```diff
@@ -15,8 +15,8 @@
  */
 
 import { getBasicClickEventParameters } from 'components/main/analytics/events/common/ga4Utils';
-import { GA_4_FIELD_LIMIT } from 'components/main/analytics/constants';
 
+export const GA_4_FIELD_LIMIT = 100;
```

Comment:
May be it is better to place this constant in `app/src/components/main/analytics/events/common/constants.js`?